ui_framework/src/components/toast/_toast.scss

```diff
@@ -81,5 +90,14 @@ $toastTypes: (
   }
 
 .kuiToastHeader--withBody {
-  margin-bottom: $kuiVerticalRhythmS;
+  margin-bottom: $kuiSizeM;
+}
+
+@include screenXSmall {
```

Comment:
And similar to my comment about `width`, above, it seems like this is more of a `kuiGlobalToastListItem` concern.